From 7aab154c74ee9a6487e38300d907bd797fd5ba0e Mon Sep 17 00:00:00 2001
From: Matthias Clasen <mclasen@redhat.com>
Date: Sun, 30 Jun 2013 01:40:26 +0000
Subject: um-crop-area.c: Relicense to GPLv2+

This makes the file fit in better with the rest of Cheese,
license-wise.

https://bugzilla.gnome.org/show_bug.cgi?id=703239
---
diff --git a/libcheese/um-crop-area.c b/libcheese/um-crop-area.c
index 8b9323d..60ca9be 100644
--- a/libcheese/um-crop-area.c
+++ b/libcheese/um-crop-area.c
@@ -4,7 +4,7 @@
  *
  * This program is free software; you can redistribute it and/or modify
  * it under the terms of the GNU General Public License as published by
- * the Free Software Foundation; either version 3 of the License, or
+ * the Free Software Foundation; either version 2 of the License, or
  * (at your option) any later version.
  *
  * This program is distributed in the hope that it will be useful,

